api/testdata/Onboard/sampleOnboardConfig.yaml

```diff
@@ -0,0 +1,12 @@
+env: Smaug
+namespaces:
+  - enable_monitoring: false
+    name: testproject
+    quota: testquota
+project_description: This is the configuration for a sample project / app to onboard
+target_cluster: Smaug
```

Comment:
Currently it doesn't seem like the `onboard` command will actually add the namespaces and groups to the cluster for deployment (i.e. the overlay `kustomization` files)